.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - run:
           name: test.py
           command: |
-            source emsdk_env.sh
+            source emsdkpy/emsdk_env.sh
```

Comment:
In general I think this sounds good - having a proper python package could be useful in various ways!

I am concerned by breaking changes, though, like moving scripts out of the toplevel where users have expected them for long time. Is there a way to avoid such breaking changes? Perhaps we could create a new repo just for the package, which would include the main emsdk repo underneath it? Or maybe symlinks could help?

Reply:
Thanks for your response. Indeed I've thought about not moving the scripts around at first, but I've not been able to do that. IIRC the issue was for installing the scripts as package_data for the Python package, which was not possible if not under a Python module. But I can give another try.